pypeit/data/standards/noao/cygob2no9.dat

```diff
@@ -0,0 +1,100 @@
+# CygOB2No9
```

Comment:
Can you please add a README file to standards/noao/ with information on where this data came from, the units, etc. Also, I agree with @feigewang comment. Only standards that are not in calspec should be added in this directory, since the calspec files are much better, i.e. they have a finer wavelength grid.

Reply:
I added that information in noao_info.txt file, and renamed the data files following the way written in aaareadme.ctio